packages/app/src/app/pages/Pro/index-v2.tsx

```diff
@@ -101,242 +227,522 @@ const Upgrade = () => {
     workspace => workspace.id === activeTeam
   );
 
-  const isPersonalWorkspace = personalWorkspaceId === activeTeam;
-  const numberOfViewers = activeWorkspace.userAuthorizations.filter(
-    ({ authorization }) => authorization === TeamMemberAuthorization.Read
-  ).length;
   const numberOfEditors = activeWorkspace.userAuthorizations.filter(
     ({ authorization }) => authorization !== TeamMemberAuthorization.Read
   ).length;
+  setSeats(numberOfEditors);
 
-  const isPersonalPro = isPersonalWorkspace || user.subscription;
-  const isTeamPro = activeWorkspace.joinedPilotAt;
-
-  let currentPlanName: string;
-  if (isTeamPro) currentPlanName = 'Team Pro';
-  else if (isPersonalWorkspace && isPersonalPro) {
-    currentPlanName = 'Personal Pro';
-  } else currentPlanName = 'Community workspace (Free)';
+  const isLegacyPersonalPro = isPersonalWorkspace && user.subscription;
```

Comment:
Existing Stripe users will become Legacy Pro. New pro users would fall in `workspace.subscription`